src/nidcpower/metadata/attributes.py

```diff
@@ -37,7 +37,7 @@
         'type': 'ViBoolean'
     },
     1050005: {
-        'access': 'read-write',
+        'access': 'read only',
```

Comment:
I believe this is a mistake. Please double check internal NI meetadata (it was incorrect for a few days)